netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
+
 %if 0%{?fedora} || 0%{?rhel} >= 7
-%global do_systemd 1
 %else
-%global do_systemd 0
+%undefine	with_systemd
 %endif
 
 Summary:	Linux real time system monitoring, over the web
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
 Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
-License:	GPLv2+
-URL:		http://firehol.org
-Source:		%{name}-@PACKAGE_VERSION@.tar.bz2
-
+License:	GPL v3+
+Group:		Applications/System
+Source0:	https://github.com/firehol/netdata/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
you shouldn't mess with field `Version:`, it must match upstream version. if you need to put something else to version tag, use `Release:` field, otherwise you end up `Epoch:` field bumping to overcame upgrade issues. 

anyway restored.